packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol

```diff
@@ -192,7 +194,7 @@ contract BullStrategyTestFork is Test {
     }
 
     function testDepositWhenTotalSupplyLessThanMinimum() public {
-        uint256 crabToDeposit = 1e12;
+        uint256 crabToDeposit = WETH_DECIMALS_DIFF;
```

Comment:
not supposed to be WETH_DECIMALS diff, should revert this change, just coincidentally 1e12